modules/update-best-practices.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_mod-docs-content-type: PROCEDURE
```

Comment:
```suggestion
:_mod-docs-content-type: REFERENCE
```

Reply:
Good catch, I copied and pasted this metadata stuff from another module